examples/rpi-pico-w/src/main.rs

```diff
@@ -227,4 +227,10 @@ impl cyw43::SpiBusCyw43 for MySpi {
         self.read(read).await;
         self.cs.set_high();
     }
+
+    async fn wait_for_event(&mut self) {}
```

Comment:
Perhaps it'd be a good idea to add a `yield_now()` in the default impl, so it doesn't become a hot loop if it's unimplemented.